code/modules/paperwork/ticketmachine.dm

```diff
@@ -172,6 +189,10 @@
 /obj/machinery/ticket_machine/attack_ai(mob/user)
 	return
 
+/obj/machinery/ticket_machine/examine(mob/user)
+	. = ..()
+	. += "<span class='info'>Use an ID card with HOP access on this machine to [disabled ? "enable":"disable"] ticket dispensing.</span>"
```

Comment:
I *think* the suggestion is to
```suggestion
	. += "<span class='info'>Use an ID card with HOP access on [name] to [disabled ? "enable":"disable"] ticket dispensing.</span>"
```
but please get a second opinion from someone who actually knows the style guide

Reply:
I will ask those in the know.

Reply:
Its a niche thing, and after talking it over with some people... the way it is now is not a problem.